packages/core/src/test/media-validation.test.tsx

```diff
@@ -1,101 +1,36 @@
-import {render} from '@testing-library/react';
-import React, {useContext} from 'react';
-import {Audio} from '../audio';
-import {Internals} from '../internals';
-import {Video} from '../video';
+import {validateMediaProps} from '../validate-media-props';
 import {expectToThrow} from './expect-to-throw';
 
-const Wrapper: React.FC = ({children}) => {
-	const compositions = useContext(Internals.CompositionManager);
-	return (
-		<Internals.RemotionRoot>
-			<Internals.CompositionManager.Provider
-				value={{
-					...compositions,
-					compositions: [
-						{
-							height: 1080,
-							width: 1080,
-							fps: 30,
-							durationInFrames: 30,
-							id: 'markup',
-							nonce: 0,
-							component: React.lazy(() =>
-								Promise.resolve({
-									default: (() => null) as React.ComponentType<unknown>,
-								})
-							),
-						},
-					],
-					currentComposition: 'markup',
-				}}
-			>
-				{children}
-			</Internals.CompositionManager.Provider>
-		</Internals.RemotionRoot>
-	);
-};
-
 test('It should not allow a video to have a negative volume', () => {
 	expectToThrow(
-		() =>
-			render(
-				<Wrapper>
-					<Video src="hi" volume={-1} />
-				</Wrapper>
-			),
+		() => validateMediaProps({volume: -1}, 'Video'),
 		/You have passed a volume below 0 to your <Video \/> component./
 	);
 });
 
 test('It should not allow an audio element to have a negative volume', () => {
 	expectToThrow(
-		() =>
-			render(
-				<Wrapper>
-					<Audio src="hi" volume={-1} />
-				</Wrapper>
-			),
+		() => validateMediaProps({volume: -1}, 'Audio'),
 		/You have passed a volume below 0 to your <Audio \/> component./
 	);
 });
 
 test('It should not allow an audio element to have a wrong type', () => {
 	expectToThrow(
-		() =>
-			render(
-				<Wrapper>
-					{/**
-           *
-           // @ts-expect-error */}
-					<Audio src="hi" volume="wrong" />
-				</Wrapper>
-			),
-		/You have passed a volume of type string to your <Audio \/> component. Volume must be a number or a function/
+		// @ts-expect-error
+		() => validateMediaProps({volume: 'wrong'}, 'Audio'),
+		/You have passed a volume of type string to your <Audio \/> component./
 	);
 });
 
-test('It should not allow an video element to have a wrong type', () => {
+test('It should not allow a video element to have a wrong type', () => {
```

Comment:
I think the previous tests were better because they were testing that e.g. the <Audio/> tag was actually calling the validateMediaProps function.
Shouldn't this in theory result in a higher code coverage as well?

Nevertheless, I am happy to leave it as is, definitely don't spend time refactoring it again!